pkg/pathmapping/tag_digest_mapping_test.go

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (c) 2019-Present Pivotal Software, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package pathmapping
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/pivotal/image-relocation/pkg/image"
+)
+
+var _ = Describe("FlattenRepoPathPreserveTagDigest", func() {
```

Comment:
`tag_digest_mapping_test.go` seems to be doing a very similar job as `path_mapping_test.go`. What am I missing?

Reply:
Nothing. ;-)  One mapping is essentially an extension of the other one, so the tests do overlap quite a bit. We could share methods between the tests, but I doubt that would make them easier to read.